llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -21962,6 +21962,35 @@ SDValue tryLowerPartialReductionToWideAdd(SDNode *N,
   return DAG.getNode(TopOpcode, DL, AccVT, BottomNode, ExtOp);
 }
 
+static SDValue foldRevInvolution(SDNode *N) {
```

Comment:
Please add a doc comment that explains what this does (I know "Involution" is the mathematical term for this, but it's not a super common term).